samples/README.md

```diff
@@ -103,6 +104,7 @@ for more detailed instructions.
 * [Get Public Access Prevention](#get-public-access-prevention)
 * [Get Requester Pays Status](#get-requester-pays-status)
 * [Get Retention Policy](#get-retention-policy)
+* [Get RPO (Recovery Point Objective)](#get-rpo)
```

Comment:
Same for L107 and L146? Let me know if you have any thoughts on streamlining this process... very much open to discussion :)